IA/IA_bag_and_tag.py

```diff
@@ -0,0 +1,79 @@
+import re
+import os
+import bagit
+import argparse
+from datacite import DataCiteMDSClient
+import settings
+import zipfile
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+
+
+def build_doi(guid):
+    return settings.DOI_FORMAT.format(prefix=settings.DATACITE_PREFIX, guid=guid)
```

Comment:
What happens if the registration hasn't been sent to DataCite for a DOI? For example, I tried [this registration](https://osf.io/j7kve) and, if I constructed the DOI properly, would have gotten [this 404](https://doi.org/10.17605/osf.io/j7kve).

It might be better to just build this data from what we have. Ember-osf-web [already does this](https://github.com/CenterForOpenScience/ember-osf-web/blob/58aa472e2854519f9ab12b17665ba5871a38863d/lib/registries/addon/overview/route.ts#L57-L71) so we have the mappings for all the mandatory fields. 

![Screen Shot 2019-11-22 at 12 50 50 PM](https://user-images.githubusercontent.com/6599111/69448548-c39d3680-0d26-11ea-8671-de3387ccf0d6.png)

![Screen Shot 2019-11-22 at 12 25 49 PM](https://user-images.githubusercontent.com/6599111/69448552-c5ff9080-0d26-11ea-9966-c8aac5eed6d6.png)

Well, other than `publication_year`, I suppose, but that should be easy to figure out.

Reply:
I believe our agreed upon solutions was to exclusively use registrations that already had DOIs, though seeing how much time remains in the sprint minting them with proper metadata is probably doable, but will need to update `identifier` field for that registration.

Reply:
What's the behavior if there's no DOI? Does it just 404 and bail out?